.sync.yml

```diff
@@ -4,54 +4,3 @@ spec/spec_helper.rb:
 spec/default_facts.yml:
   extra_facts:
     path: "/bin:/usr/bin:/usr/local/bin"
-.gitlab-ci.yml:
```

Comment:
Yes, that's why I asked if you had some kind of up-to-date template to integrate to the modulesync_config repo to manage the `gitlab-ci.yml` file with modulesync. You said you don't really have one and that we can just ignore this file.

So this in modulesync_config, we stopped managing `.gitlab-ci.yml`:
https://github.com/ccin2p3/modulesync_config/blob/642cf13bf00b4e55022b2da81c916177b50cea78/config_defaults.yml#L124-L126

Because this file is not managed anymore, this module specific config for an unmanaged file can be removed.